frodo.js

```diff
@@ -1,66 +1,67 @@
-/**
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?